facade/service.go

```diff
@@ -680,7 +679,6 @@ func (f *Facade) RestoreServices(ctx datastore.Context, tenantID string, svcs []
 	traverse = func(parentID string) error {
 		for _, svc := range svcsmap[parentID] {
 			alog := f.auditLogger.Message(ctx, "Restore Service").WithField("servicename", svc.Name).Action(audit.Restore).Entity(&svc)
-			svc.DatabaseVersion = 0
```

Comment:
Similar to Pool, this may be an issue during restore

Reply:
yes you're right, there was an issue with restoring